npm/cheerio/1.0.0/index.n4jsd

```diff
@@ -0,0 +1,43 @@
+/**
+ * @see https://github.com/cheeriojs/cheerio
+ * 
+ * TODO incomplete
```

Comment:
Please mention the package version and elaborate what is the `TODO` here.

Reply:
removed the TODO since all n4jsd lack accuracy in detail